src/Files.App/Dialogs/ReleaseNotesDialog.xaml.cs

```diff
@@ -68,6 +68,7 @@ private ContentDialog SetContentDialogRoot(ContentDialog contentDialog)
 
 		private async void BlogPostWebView_CoreWebView2Initialized(WebView2 sender, CoreWebView2InitializedEventArgs args)
 		{
+			BlogPostWebView.CoreWebView2.Profile.PreferredColorScheme = (CoreWebView2PreferredColorScheme)RootAppElement.RequestedTheme;
```

Comment:
TODO: PreferredColorScheme uses 'Auto' instead of 'Default', not sure if casting works in that case
It didnt cause problems, but i would like to check this again